.github/workflows/c-cpp.yml

```diff
@@ -181,9 +180,14 @@ jobs:
 
     outputs:
       BuildTag: ${{ env.build_variable }}
+      ShortSHA: ${{ env.git_short_sha }}
 
   create-release:
     needs: [build]
+    strategy:
+      fail-fast: false
+      matrix:
+        repo: [master, store]
```

Comment:
Are you sure about disabling fail-fast on the release build? Won't that mess up the auto-updater for an OS if the build for this OS fails?

Reply:
This is so that if one job fails, the parallel workflow is not aborted. (Here is master and store) This means that if one release fails, although I don't think that will happen, it will run until the other completes.
Also, since the update script only references the master repo, I don't think it will cause any problems.